spynnaker/pyNN/utilities/utility_calls.py

```diff
@@ -308,22 +308,20 @@ def _validate_mars_kiss_64_seed(seed):
     return seed
 
 
-def create_mars_kiss_seeds(rng, seed=None):
+def create_mars_kiss_seeds(rng):
     """ generates and checks that the seed values generated by the given\
         random number generator or seed to a random number generator are\
         suitable for use as a mars 64 kiss seed.
 
     :param rng: the random number generator.
-    :type rng: None or ~numpy.random.RandomState
+    :type rng: ~numpy.random.RandomState
     :param seed:
         the seed to create a random number generator if not handed.
     :type seed: int or None
     :return: a list of 4 ints which are used by the mars64 kiss random number
         generator for seeds.
     :rtype: list(int)
     """
-    if rng is None:
-        rng = numpy.random.RandomState(seed)
```

Comment:
I was a bit worried about taking this out because I was sure that rng=None was being passed in here, however I've just noticed it never is - the confusion is because in the init of SpikeSourcePoissonVertex self.__rng is set to None at line 132, but then later on line 255 (still in the init) it is actually set to a value.  I'd suggest that line 132 should be removed from that file to avoid any further confusion in the future.

Reply:
Agreed - fixed!